File: LeetCode/C#/88. Merge Sorted Array.cs
public class Solution {
    public void Merge(int[] nums1, int m, int[] nums2, int n) {
        if(nums1.Length == 1)
        {
            if(nums1[0] == 0)
            {
                nums1[0] = nums2[0];
            }
        }
        else{
            for(int i = m, j = 0; i < nums1.Length; i++, j++)
            {
                nums1[i] = nums2[j];
            }
            
            Array.Sort(nums1);
        }
    }
}

File: LeetCode/Top Interview 150 - Java/88. Merge Sorted Array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int p1 = m - 1;
        int p2 = n - 1;
        int target = m + n - 1;

        while(p1 >= 0 && p2 >= 0) {
            if(nums1[p1] > nums2[p2]) {
                nums1[target] = nums1[p1];
                p1--;
            }
            else {
                nums1[target] = nums2[p2];
                p2--;
            }
            target--;
        }

        while(p2 >= 0) {
            nums1[target--] = nums2[p2--];
        }
    }
}